#[
error: Imported raise!
Traceback:
    from raise_imported.lly:9: in __main__
]#

import ex

raise ex.error("Imported raise!")
